docs/project-structure.md

```
@@ -0,0 +1,83 @@
+# Project Structure
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)